README.md

```diff
@@ -1,2 +1,124 @@
-# doogie_robot
-Robot packages for Doogie Mouse
+# **doogie_robot**
+
+The **doogie_robot** stack provides a [ROS] interface communication to Doogie Mouse **hardware** plataform. For tutorials, please check [doogie_robot/Tutorials] on wiki.
+
+**Keywords:** Micromouse, Gazebo, ROS
+
+**Author: [Mateus Meneses]<br />
+Affiliation: [BIR - Brazilian Institute of Robotics]<br />
+Maintainer: Mateus Meneses, mateusmenezes95@gmail.com **
+
+### Supported Versions
+
+- **Kinetic**: Built and tested under [ROS] Kinetic and Ubuntu 16.04
+
+<!-- [![Build Status](http://rsl-ci.ethz.ch/buildStatus/icon?job=ros_best_practices)](http://rsl-ci.ethz.ch/job/ros_best_practices/) TODO -->
+
+### Dependencies 
+- [doogie_base] : ROS packages stack with all common packages for working with Doogie Mouse
+
+<p align="center">
+   <img src="docs/doogie_robot.jpeg" alt="doogie_robot hw" title="Example Image">
+</p>
+</br>
+
+</br>
+
+# **Table of Contents**
+- [**doogie_robot**](#doogierobot)
+    - [Supported Versions](#supported-versions)
+    - [Dependencies](#dependencies)
+- [**Table of Contents**](#table-of-contents)
+- [**File System**](#file-system)
+- [**Installation**](#installation)
+    - [Building from Source:](#building-from-source)
+  - [Example of Usage](#example-of-usage)
+    - [Launch Doogie Mouse at Gazebo:](#launch-doogie-mouse-at-gazebo)
+- [**Purpose of the Project**](#purpose-of-the-project)
+- [**License**](#license)
+- [**Bugs & Feature Requests**](#bugs--feature-requests)
+
+# **File System**
+
+- [doogie_bringup] : Package with launch files to bringup Doogie Mouse robot controllers.
```

Comment:
I think that in bringup will launch other stuff too, such as IR sensors driver node and button command node. Maybe we can describe here that the codes for robot control are in this package too